### PR TITLE
feat(callhome): add bustype and rotational properties to block device

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added bus type("connection_type") and rotational ("is_rotational") properties to block device to be able to generate storage media metrics.

connection_type: the type of bus through which the device is connected to the system
is_rotational: indicates whether the device is rotational or non-rotational

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
Please don't smart link PR's on the PR description nor commits

File: control-plane/stor-port/src/types/v0/transport/blockdevice.rs [hunk 1]
```diff
@@ -78,19 +78,23 @@ pub struct BlockDevice {
     /// identifies if device is available for use (ie. is not "currently" in
     /// use)
     pub available: bool,
+    pub connection_type: String,
```
Comment (tiagolobocastro) [inline control-plane/stor-port/src/types/v0/transport/blockdevice.rs:82]: missing doc comments

**Review by tiagolobocastro — APPROVED** [review]

File: control-plane/stor-port/src/types/v0/transport/blockdevice.rs (hunk 1 above)
Comment (datacore-vvarakantham) [reply control-plane/stor-port/src/types/v0/transport/blockdevice.rs:82]: Added the doc comments now.

**Review by tiagolobocastro — APPROVED** [review]

**Review by sinhaashish — APPROVED** [review]

**Comment by datacore-vvarakantham** [comment]
bros merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**